apps/cli/src/commands/device/update.ts

```diff
@@ -0,0 +1,27 @@
+// eslint-disable-next-line import/order
```

Comment:
If we don't add this, eslint will change the order of import as provided in the eslint config file. We don't want that to happen here. Since `config` should be imported before importing anything else to set the `process.env` variables.